src/App.js

```diff
@@ -5,25 +5,42 @@ import './App.css';
 import posts from './api/posts';
 import comments from './api/comments';
 import users from './api/users';
+import PostList from './components/PostList/PostList';
 
-const App = () => (
-  <div className="App">
-    <h1>Static list of posts</h1>
-
-    <p>
-      <span>posts: </span>
-      {posts.length}
-    </p>
+function groupPostsWithUsersAndComments(postList, commentList, userList) {
+  return postList.map(post => (
+    {
+      ...post,
+      user: userList.find(user => user.id === post.userId),
```

Comment:
will be great to create usersMap, to prevent iterate users array for each post

Same with comments